safrole/tiny/publish-tickets-no-mark-3.json

```diff
@@ -1,422 +1,415 @@
             },
             {
-                "ed25519": "0x22351e22105a19aabb42589162ad7f1ea0df1c25cebf0e4a9fcd261301274862",
                 "bandersnatch": "0x3d5e5a51aab2b048f8686ecd79712a80e3265a114cc73f14bdb2a59233fb66d0",
+                "ed25519": "0x22351e22105a19aabb42589162ad7f1ea0df1c25cebf0e4a9fcd261301274862",
                 "bls": "0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
                 "metadata": "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
             },
             {
-                "ed25519": "0xb3e0e096b02e2ec98a3441410aeddd78c95e27a0da6f411a09c631c0f2bea6e9",
                 "bandersnatch": "0x7f6190116d118d643a98878e294ccf62b509e214299931aad8ff9764181a4e33",
+                "ed25519": "0xb3e0e096b02e2ec98a3441410aeddd78c95e27a0da6f411a09c631c0f2bea6e9",
                 "bls": "0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
                 "metadata": "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
             }
         ],
-        "tickets_accumulator": [
+        "gamma_a": [
             {
-                "id": "0x0a197e5056b76b3b37c0815a4e1ac81044b3baca6a74bdc1ea4f57931256639d",
-                "attempt": 0
+                "id": "0x09a696b142112c0af1cd2b5f91726f2c050112078e3ef733198c5f43daa20d2b",
+                "attempt": 1
             },
             {
-                "id": "0x4252603fa8b584704528da6dace299a05cb6891e4c59260e8d2fea39053db84c",
+                "id": "0x3a5d10abc80dda33fe3f40b3bb2e3eefd3e97dda3d617a860c9d94eb70b832ad",
                 "attempt": 1
             },
             {
-                "id": "0xd4340dddd7896ae9c5ee9b822bec9e29693dceff5b4fa642aa5462ffd27876b1",
+                "id": "0x5d91e5951d2b62c424b767220498e6d64280fc84ffb16951d93a00c009341498",
                 "attempt": 0
             },
             {
-                "id": "0xf2f2bc02763b99882bf268731c30c6271269e1c64f43b996954daaee1c3f9fbf",
-                "attempt": 1
+                "id": "0xf28c553c54ea497bb39e20344e7a687523d06390bb8a24a919183599e584e813",
```

Comment:
(this comment is supposed to be on the `full` equivalent of this file, but the value is the same).

This is above my calculated ticket score threshold with SAFROLE parameters `r=2, s=600, a=2, v=1023` the cutoff is about `0x9623 << 240`.

Reply:
I'm not sure I understood your comment.
There is not threshold in Safrole. The threshold was in Sassafras (to allow the user to fine tune the params).

Reply:
Here you submit your tickets as far as their score/id end up being in the state after the STF execution.
The rule when gamma_a is full, is that tickets with smaller score win. 
Tickets already in the state with bigger value are discarded.
 All new submitted tickets must be "consumed" (i.e. none of the new tickets is allowed to lose)

Reply:
No problem: https://github.com/w3f/jamtestvectors/pull/5/commits/86c007a1e3a11da76c8b8f1f34a4e20992606c27